tools/rpkg/R/register.R

```diff
@@ -36,3 +36,27 @@ duckdb_unregister <- function(conn, name) {
   invisible(TRUE)
 }
 
+#' @rdname duckdb_register
+#' @export
+duckdb_register_arrow <- function(conn, name, arrow_table) {
+  stopifnot(dbIsValid(conn))
+  record_batch_reader <- arrow::Scanner$create(arrow_table)$ToRecordBatchReader()
```

Comment:
```suggestion
```

Looks like this is covered in `export_fun()`

Reply:
Indeed this is not necessary there.